tests/utils/compare.js

```diff
@@ -32,6 +32,7 @@ function sendReference (filename, data) {
 }
 
 function resetFile(pdfFile) {
+  pdfFile = pdfFile.replace(/%PDF-(.*)(\n|\r\n|\r)%(.*)\n/, '%PDF-$1$2');
```

Comment:
Why?

Reply:
Anyway, it had nothing to do with the template system so I removed that change...